### ISSUE TITLE
python import statements not working when called from python-shell

### ISSUE THREAD

**Issue by Nirvan101** [extrabacon/python-shell#102]
from preprocessor import pad_vec_sequences, labels
    from keras.models import load_model
    import spacy
    from preprocessor import nlp
    import numpy as np
    import sys
    from dependency_tree import to_nltk_tree , to_spacy_desc

    print('Hi')`

This code works when I run it from terminal. I have all the mentioned libraries installed. I'm using python 3.5.

I'm now trying to call this python file from `python-shell` in node-js . But I'm getting errors for the import statements saying import modules can't be found. I have made sure that python3.5 is used in both cases- from terminal and through nodeJs . The `pythonPath` I mentioned in `options` I got from `sys.executable`

    app.post( '/api/message', function(req, res) {

    PythonShell.run('demo.py', {
        args: [req.body.input.text], pythonPath : '/usr/bin/python3'
    }    , function (err, results) {
    console.log(results);
      return res.json({
      'input' : { 
          'text' : ''
              },
        'output': {
            'text': results //+
            //'<img src="http://localhost:8080/Finance/image1.jpg" class="imageclass">'
         }    
    })
    });
    })

I don't understand why this is happening. There are no import errors when I run from terminal, but when I run from node-js I get import errors.  `numpy` and `sys` are fine, the others are giving errors.

**Comment by Almenon** [comment]
set req.body.input.text to help("modules").  When that python runs as a result you will get a list of modules you have.

How did you install these modules? with pip install?

As an alternative to python-shell you might be able to use [python-bridge](https://www.npmjs.com/package/python-bridge)

**Comment by Almenon** [comment]
Are you still having the problem? If not I'll close the issue.
